src/applications/vaos/utils/appointment.js

```diff
@@ -306,16 +306,35 @@ function getAppointmentDuration(appt) {
   return isNaN(appointmentLength) ? 60 : appointmentLength;
 }
 
+const ICS_LINE_LIMIT = 74;
+function formatDescription(description) {
```

Comment:
@narin Added this after your review. ICS files have formatting requirements for fields over 75 characters, so I had to add this to get the ICS to be valid and show up correctly in iCal.